RevenueCatUI/Data/PaywallViewMode.swift

```diff
@@ -13,4 +13,26 @@ public enum PaywallViewMode {
     /// Paywall is displayed full-screen, with as much information as available.
     case fullScreen
 
+    /// Paywall is displayed with a square aspect ratio. It can be embedded inside any other SwiftUI view.
+    case square
```

Comment:
I'd also like this be renamed to like `card`? 🤔 `square` feels a little to concrete on the design and not the use case 🤷‍♂️

Reply:
I love `card` 👌🏻